ivre/parser/iptables.py

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env python
+
+# This file is part of IVRE.
+# Copyright 2011 - 2018 Pierre LALET <pierre.lalet@cea.fr>
+#
+# IVRE is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# IVRE is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with IVRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# Maintainers:
+#   - 2018 Francois CHENAIS <francois.chenais@cea.fr>
+#
+
+"""Support for Iptables log from syslog files
+
+
+Feb  4 05:30:11 pi01 kernel: [3240403.495065] IPTABLES/UDP/IN=enxb827eb8f8a4f OUT= MAC=ff:ff:ff:ff:ff:ff:18:62:2c:7e:45:d0:08:00:45:00:00:ec:00:00:40:00:40:11:df:e3 SRC=192.168.0.254 DST=192.168.0.255 LEN=236 TOS=0x00 PREC=0x00 TTL=64 ID=0 DF PROTO=UDP SPT=138 DPT=138 LEN=216
```

Comment:
I think the prefix (`IPTABLES/UDP/IN=`) should not be hard-coded but somehow configurable.

Reply:
The prefix depends of the iptables conf  --log-prefix. In this case the log prefix is 'IPTABLES/UDP/' as an example. Because the log isn't separate from the first field IN=, I use the find at the begin of the parse_line function.
It should be safer to find the last occurancy if the prefix contains IN=...:P

Reply:
Yep I understand that. Maybe we could by default try to match any iptables log (without caring about the prefix) and propose an option to look for a specific prefix?